framework/core/js/src/common/Translator.tsx

```diff
@@ -88,4 +96,22 @@ export default class Translator {
 
     return id;
   }
+
+  /**
+   * Formats the time.
+   *
+   * The format of the time will be chosen by the following order:
+   * - Custom format defined in the item list.
+   * - The format defined in current locale.
+   * - DayJS default format.
+   */
+  formatDateTime(time: Dayjs, id?: string): string {
+    const format = id && (this.translations[id] ?? id);
+    const formatCallback = id && this.dateTimeFormats.has(id) && this.dateTimeFormats.get(id);
+    if (formatCallback) {
+      const result = formatCallback(format);
```

Comment:
maybe add a `try catch` block?

just advice, no force.

Reply:
We don't have to help bad extensions clean up their mess, like what `extend`/`override` did. Even if you do catch the error, what are you gonna do with it, make it silently fail?